assets/.eslintrc.json

```diff
@@ -1,5 +1,8 @@
 {
   "env": {
     "browser": true
+  },
+  "rules": {
+    "strict": "off"
```

Comment:
I think it's more to do with the Google Analytics code at the bottom of the script?